zh/releases/release-1.1.0-rc.1.md

```diff
@@ -8,56 +8,59 @@ Release date: April 1, 2020
+- 在 `Restore` 中使用 `tidb-lightning` 替代 `loader` ([#2068](https://github.com/pingcap/tidb-operator/pull/2068), [@Yisaer](https://github.com/Yisaer))
+- 为 TiDB 组件添加 `cert-allowed-cn` 支持 ([#2061](https://github.com/pingcap/tidb-operator/pull/2061), [@weekface](https://github.com/weekface))
+- 修复 PD `location-labels` 配置项的问题 ([#1941](https://github.com/pingcap/tidb-operator/pull/1941), [@aylei](https://github.com/aylei))
+- 可以通过 `spec.paused` 控制 TiDB 集群暂停部署 
+([#2013](https://github.com/pingcap/tidb-operator/pull/2013), [@cofyc](https://github.com/cofyc))
+- 在使用 CR 部署 TiDB 集群时，使 TiDB 的 `mac-backups` 配置项默认值设为 `3` ([#2045](https://github.com/pingcap/tidb-operator/pull/2045), [@Yisaer](https://github.com/Yisaer))
+- 让一些组件的配置项可以通过环境变量的方式来指定  ([#2052](https://github.com/pingcap/tidb-operator/pull/2052), [@cofyc](https://github.com/cofyc))
+- 修复 `kubectl get tc` 不能显示正确镜像的问题 
+([#2031](https://github.com/pingcap/tidb-operator/pull/2031), [@Yisaer](https://github.com/Yisaer))
+- 在 `spec.tikv.maxFailoverCount` 及 `spec.tidb.maxFailoverCount` 未定义时，将其默认值设为 `3` ([#2015](https://github.com/pingcap/tidb-operator/pull/2015), [@Yisaer](https://github.com/Yisaer))
+- 在 `maxFailoverCount` 设为 `0` 时禁用自动故障转移的功能  ([#2015](https://github.com/pingcap/tidb-operator/pull/2015), [@Yisaer](https://github.com/Yisaer))
+- 支持通过 Terraform 使用TidbCluster 及 TidbMonitor CR 在 ACK 上部署 TiDB 集群  ([#2012](https://github.com/pingcap/tidb-operator/pull/2012), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- 将 TidbCluster 中的 PDConfig 升级到 PD v3.1.0 ([#1928](https://github.com/pingcap/tidb-operator/pull/1928), [@Yisaer](https://github.com/Yisaer))
+- 支持通过 Terraform使用 TidbCluster 及 TidbMonitor CR 在 AWS 上部署 TiDB 集群 ([#2004](https://github.com/pingcap/tidb-operator/pull/2004), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- 将 TidbCluster 中的 TidbConfig 升级到 TiDB v3.1.0 ([#1906](https://github.com/pingcap/tidb-operator/pull/1906), [@Yisaer](https://github.com/Yisaer))
+- 允许用户在 TiDB 初始化作业时为 initContainers 指定资源 
+([#1938](https://github.com/pingcap/tidb-operator/pull/1938), [@tfulcrand](https://github.com/tfulcrand))
+- 为 Pump 及 Drainer 添加 TLS 支持 ([#1979](https://github.com/pingcap/tidb-operator/pull/1979), [@weekface](https://github.com/weekface))
+- 为 auto-scaler 和 initializer 添加文档与示例 ([#1772](https://github.com/pingcap/tidb-operator/pull/1772), [@Yisaer](https://github.com/Yisaer))
+    - 添加检查以保证当 TidbMonitor 的 serviceType 为 NodePort 时，NodePort 不会被改变
+    - 添加 EnvVar 排序来避免模拟器从同一份 TidbMonitor 规范渲染出不同的结果
+    - 修复 TidbMonitor LoadBalancer IP 不被使用的问题 ([#1962](https://github.com/pingcap/tidb-operator/pull/1962), [@Yisaer](https://github.com/Yisaer))
+- tidb-initializer 支持 TLS ([#1931](https://github.com/pingcap/tidb-operator/pull/1931), [@weekface](https://github.com/weekface))
+    - 修复 Advanced StatefulSet 不能与 webhook 工作的问题
```

Comment:
where is the issue link?

Reply:
Ah, the English version didn't provide a link.

Reply:
ok, it's 1963